src/db/clients/postgresql.js

```diff
@@ -358,25 +370,31 @@ const getSchema = async (connection) => {
   return result.rows[0].schema;
 };
 
-export const truncateAllTables = async (connection) => {
-  const schema = await getSchema(connection);
-  const sql = `
-    SELECT quote_ident(table_name) as table_name
-    FROM information_schema.tables
-    WHERE table_schema = '${schema}'
-    AND table_type NOT LIKE '%VIEW%'
-  `;
-  const [result] = await executeQuery(connection, sql);
```

Comment:
The reason i that i wanted to use parameters instead of trying to escape the schema which is inefficient and a bit less secure.